src/config/type.ts

```diff
@@ -0,0 +1,5 @@
+export enum GroupByInterval {
```

Comment:
Do you think `DateRange` sounds better?
Rename the file given the name of the enum.